pkg/metrics/statsd/cosmosdb/metrics.go

```diff
@@ -0,0 +1,77 @@
+package cosmodb
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Azure/ARO-RP/pkg/metrics"
+)
+
+var actions = map[string]bool{
+	"colls":    true,
+	"docs":     true,
+	"triggers": true,
+	"dbs":      true,
+}
+
+var tables = map[string]bool{
```

Comment:
would like to avoid the need for tables like this

Reply:
lets sort https://github.com/jim-minter/go-cosmosdb/pull/2 this first and later this one